charts/taraxa-node/templates/consensus-node.yaml

```diff
@@ -44,8 +44,16 @@ spec:
               subPath: entrypoint.sh
         {{- end }}
         - name: config-adapter
+          {{- if and .Values.consensusnode.image.repository .Values.consensusnode.image.tag }}
+          image: "{{ .Values.consensusnode.image.repository }}:{{ .Values.consensusnode.image.tag }}"
+          {{- else }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          {{- end }}
+          {{- if .Values.consensusnode.image.imagePullPolicy }}
```

Comment:
Wrong key. Should be `.Values.consensusnode.image.pullPolicy`

Reply:
fixed